packages/block-library/src/cover/edit.js

```diff
@@ -156,7 +156,7 @@ function ResizableCover( {
 	);
 }
 
-function onCoverSelectMedia( setAttributes ) {
+export function onCoverSelectMedia( setAttributes ) {
```

Comment:
I think this is OK, but I'm wondering if it'd be better to extract this logic to a shared `utils` file instead, and have a `attributesFromMedia` function there.

Reply:
I think it is better to extract it, especially since the web file is kinda big already. https://github.com/WordPress/gutenberg/pull/19722/commits/a4dfbdd5911e0b354d75cc32169b218c14e3df0f thanks for the name suggestion!